app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::UsersController < ApplicationController
+	before_action :authenticate_user!
+
+	def index
+		@users = User.where(role: User.roles[:user])
```

Comment:
Can we move it into a scope eg, `by_role` and it takes `role` parameter?

Reply:
I made it into a namespace after consulting with Talha Javed bhai and Miss Ayza as well. But if u still think there is a need to change it then sure, just let me know.